src/app/ApplicationTemplate.UWP/Views/Content/Authentication/CreateAccountPage.xaml

```diff
@@ -44,19 +143,67 @@
 								 x:Uid="CreateAccount_Password"
 								 PlaceholderText="Password"
 								 Password="{Binding Form.Password, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
+								 ios:ReturnKeyType="Next"
+								 android:ImeOptions="Next"
+								 ue:PasswordBoxBehavior.NextControl="{Binding ElementName=ConfirmPasswordField}" />
+				</local:DataValidationView>
+
+				<!-- Comfirm Password -->
+				<local:DataValidationView Model="{Binding Form}"
+										  PropertyName="ConfirmPassword"
+										  Margin="0,4">
+					<PasswordBox x:Name="ConfirmPasswordField"
+								 x:Uid="CreateAccount_ConfirmPassword"
+								 PlaceholderText="Confirm Password"
+								 Password="{Binding Form.ConfirmPassword, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}"
 								 ios:ReturnKeyType="Done"
 								 android:ImeOptions="Done"
 								 ue:PasswordBoxBehavior.DismissKeyboardOnEnter="True"
 								 ue:PasswordBoxBehavior.EnterCommand="{Binding CreateAccount}" />
 				</local:DataValidationView>
 
+				<!-- Top 3 Favorite Donnut -->
+				<local:DataValidationView Model="{Binding Form}"
+										  PropertyName="FavoriteDonuts"
+										  Margin="0,4">
+					<StackPanel>
+
+						<!-- Label -->
+						<TextBlock Text="Favorite Donuts (pick at least 3)"
+								   Foreground="Black"
+								   FontSize="16" />
+						<TextBlock Text="{Binding Form, Converter={StaticResource debug}, Mode=TwoWay}"
+								   Foreground="Black"
+								   FontSize="16" />
+
+						<!-- List -->
+						<ListView ItemsSource="{Binding Donuts}"
```

Comment:
FavoriteDonuts is always null for some reason. Debug converter on Form shows that's null and when breaking into the validation rule show that its null